### ISSUE TITLE
Finalizing airspace

### ISSUE THREAD

**Issue by 132ndNeck** [132nd-vWing/TRMA#15]
We have now conducted two test events ( #14 ) and recived initial inputs to ranges and airspace prior to the Kola map beeing released ( #1 ).
Before any other work can be done on the TRMA we need to finalize and decide on the airspace and ranges to be used. This is important to have thought out before starting to add threat sites, targets, tankers, AWACS, frequencies, , scripts, aircraft on the actual map we will use. It is important that we are thinking on the wing as a whole when making the ranges/airspace, and that we think about the complete .miz when suggesting changes. So when providing feedback and suggestions, please make sure to recommend suggestions that take this into account.

**Process:**
1. Decide on ranges and airspace layout
2. Decide on tanker tracks
3. Decide on AWACS tracks
4. Create radio presets
5. Start populating .miz with jets with correct callsigns, presets, datalink settings etc
6. Start populating .miz with needed supporting assets (Carrier, tankers, AWACS, tacans, etc)
7. Start populating .miz with scripts (carrier, tankers, AWACS, IADS, )
8. Start populating ranges with content (targets, threats, scripts)

**Current suggestion:**
![TRMA airspace suggestion v2 1](https://github.com/132nd-vWing/TRMA/assets/26002570/9b7dff56-28cf-40e5-b0f6-d1d038aea6a0)

**Mission file**
.miz file for testing:
https://cloud.132virtualwing.org/s/wa5A98stRjrdrpb

**Design pilosophy**
Here are the thoughts behind the ranges and airspace suggested.
- As much of units as possible will be deactivated by default. Users will need to activate the intended content using the F10 menu to a even larger extent than in the current ATRM. This in order to ensure we have as low impact on performance as possible
- Goal is to centralize activity at Bodø airfield as much as possible as the 132nd home base. This to increase activity and enjoyment for the 176th and for everyone to have activity while we fly. We have seen on ATRM that a split in many airbases and FOBs have had a negative impact. With making TRMA we have a way of mitigating this and having ranges and airspace that facilitate basic trainings from the same area
- For advanced scenarios and more advanced trainings there will still be oppertunities for especially AH-64 and A-10s as the slow movers to operate from  FOBs forward located closer to a FLOT (Evenes for scenario range North), and Kiruna or Jokkmokk roadbase for scenario ranges Center and South
- Several A-G ranges are located close to Bodø TMA in order to support basic A-G training for all squadrons
- Basic A-G range are located within TMA to reduce time for transit during very basic trainings
- Specific RW ranges are located in the TMA and just south of TMA in order to faciliatate the slowest mover in the wing
- A-A with drones (AI) are intended in several ranges over the ocean
- A Basic SEAD/DEAD range are placed to the south. This range can also be combined with other ranges and airspace in the south to create a advanced training range that include ocean for scenarios with ships.
- Range naming convention is not decided upon, but I suggest that we have a subnaming convention for specific target areas/scenarios to simplify For example Scenario range Center can be range 21. Then Range 21A can be CAS area along the FLOT, Range 21B can be deep strike area in the rear (simplified example). Open for suggestions and inputs

**Comment by hbjonsson** [comment]
[Kola test v2.1_j.zip](https://github.com/user-attachments/files/15521137/Kola.test.v2.1_j.zip)
Hello. I made some slight tweaks in the updated miz i attached, namely adding a slight buffer between the ranges as shown for transit corridors, allowing flow in and out of ranges without having to encroach on a potentially active range. This can also aide in scenario design as well for a potential ACO. 

![image](https://github.com/132nd-vWing/TRMA/assets/25992707/80d17b3f-ec2c-4fc3-ae4e-fe6396c6b7bf)

Extending the SEAD/DEAD range slightly (20nm) east to allow for more depth, i´d imagine this range will be used quite allot for strike training as well. This then also covers Tarnaby airport site which can be used as a potential target, it is covered by mountains and valleys, allowing for potential low altitude training.

![image](https://github.com/132nd-vWing/TRMA/assets/25992707/93465fde-75f8-4c6f-87b8-4d3c7e7b1ec2)

**Comment by hbjonsson** [comment]
There is no airport at that location ( close to Hemavan ) but it does still present a potentially good target area.
![image](https://github.com/132nd-vWing/TRMA/assets/25992707/d0963f71-66e8-4a0c-8201-a91e399e08b0)

**Comment by 132nd-Thatch** [comment]
•	Bodo, I was given taxi instructions but I had no airfield chart. We need a chart showing the taxiway letters even if they don’t show on the actual map. Each HAS needs a number and a map, merge the two.
•	Took off from Bodo and it took 15 minutes to get to the NE edge of the SEAD/DEAD range. 3 bags and I returned with 7000lbs of fuel.
•	I flew down the eastern side and although I got DE and BB on the RWR they weren’t locking me up.
•	This area is very baron and although I spotted one are in the NE corner with a bridge my connection to the PC was lost and I lost the co-ords. It was from where I cut the NE corner heading NW.
•	I flew half-way between the SEAD/DEAD range and the A/G range and had no RWR warnings.
•	Approaching the SA-2 I was getting warnings at 20NM, the same with the SA-6. Both worked well and hit me multiple times. Very easy to see the launch from their position, good training aid.
•	Approached the SA-15 and was locked up 30 miles, fired 2 x HARM and both were intercepted. I was hit multiple times.  SA sites well visible in FLIR.
•	I think this range could be extended east and that area could be used for something. Possibly tanks positioned around a valley for A/G munitions.
•	We need a tanker track down there as I think one bag would be used up extremely quickly.
•	Nice easy transit to and from this range.
•	Returned to Bodo, selected TCN 45X then CRS 070 but it didn’t line up with the runway. Probably offset by 5-10 degrees. Using MAGNETIC or TRUE?
•	Need a list on how to pronounce the name of places phonetically, I know Bodo is Buda but the others are new to me.
•	I landed, re-armed (Not on map but needs to be marked) and my connection went so I couldn’t get to RANGE 5.  Will try off-line this week.

![Screen_240531_200748](https://github.com/132nd-vWing/TRMA/assets/155889063/fc46db64-f1b2-4273-801c-713692a90c8a)
![Screen_240531_201043](https://github.com/132nd-vWing/TRMA/assets/155889063/1e413f1d-d06d-4c91-aa13-c87dfa891259)

**Comment by 132nd-Erro** [comment]
Apologies for my unavailability to join the extra test events for Kola, I've mostly had standard times available this past month. Instead, to get to experience the area I have checked out the Kola test mission in SP at other times, in between when times allow. 

Cutter and I have had a look at the proposed range geometry and we have some suggestions which will greatly improve the utilization of the designated areas, and increase the number of scenarios for the RW TRP which we can add. 

In total, they may appear as minor details, but they will have a huge impact on our utilization of the ranges. Depending on our flight mode (high level tactics or low level tactics), the available terrain features in the range plays a huge role. More so than fixed wing airframes, I would wager (but not conclude :-)) .

The Kola 2.1 test miz is the base version for suggested changes to the RW ranges. Blue A/C and their routes outline the original range geometries, whereas the DRAW objects and red A/C outline the suggested changes. 

[Kola test v2.2_Erro.zip](https://github.com/user-attachments/files/15524610/Kola.test.v2.2_Erro.zip)

![image](https://github.com/132nd-vWing/TRMA/assets/101829260/97417220-ed6f-4766-9830-6f1fa020db12)


### RW Range 1
It was sifted ~7 km to the west to:
1. facilitate ingress and use of the range area from the west and north, as opposed to mostly from the north,
2. allow a valley at the border of RW R1 and RW R2 to be part of RW R2, as the valley entrance is inside R2 and would be useful for scenarios using R2 for ingress.  The range border is now mostly through high mountainous terrain / peaks, which minimizes the inaccessible terrain in both ranges. 

RW R1 was also increased in width towards the west by ~5 km, to incorporate coastal terrain with roads and villages. In total, it extends almost 12 km further west than originally.  The mountainous terrain in the middle can be useful for lessons on tactical terrain movement, whereas the coastal areas with roads will be useful for lessons on movement to contact. 

### RW Range 2
This range was increased in witdth to the west by ~7 km to incorporate a valley which was previously in RW R1, and further south into the A-G range (in otherwise mountainous and uninhabited areas) to fully incorporate a valley from Moldjord through Storjord and further south. 

The former valley has approaches from the north and east, such that it can be useful for scenarios to coordinate time on target from multiple angles, e.g., in flight escort scenarios. The latter valley will be useful for drills on reaction to contact (not the lesson itself, but continuation training on proficiency). 

### RW Range 3
This was increased in size to the south by ~5.5 km, in order to incorporate islands in the south-west area of the range, and Jektvika in the south-center. Additionally, it was slightly modified to the east to incorporate a valley lead suitable for ingress.

**Comment by Shadoga** [comment]
Linking feedback here as well, to increase chances it doesn't get lost:
https://github.com/132nd-vWing/TRMA/issues/14#issuecomment-2143649624

**Comment by 132ndNeck** [comment]
@132nd-Erro 
Could it be an option to extend the RW range 2 further EAST?

**Comment by 132nd-Erro** [comment]
@132ndNeck Yeah we can do this if you don't want overlap with R5. Based on your input I redrew some of the RW range borders. 
However, I hope RW range 3 can be a little bit bigger than you initially proposed, roughly 5 km more to the south as described in the previous comment (and included in the attached miz/screenshot in this comment), since it is already overlapping with another range. RW range 3 is where we would like to have certain unit types for various advanced scenarios (such as SHORAD) since it is outside of the TMA. This includes an AR scenario with a DEAD component and an MQC scenario.

[Kola test v2.1_Erro-2.zip](https://github.com/user-attachments/files/15526667/Kola.test.v2.1_Erro-2.zip)
![image](https://github.com/132nd-vWing/TRMA/assets/101829260/118d9225-bd9c-4ec2-aec7-e39bd3525731)

I merged the old RW range 1 with a fraction of the old RW range 2, then re-drew a new RW range 2 to the east based on your suggestion. There might be red units here, but not in danger of ENBO traffic. The departure cone continues eastwards north of the range border. There will be live fire in RW range 1 and 2, but oriented longitudinally and/or towards the south.

Altitude restrictions can be defined by default (e.g., 3000 ft), but we would like to have the ability to request GND-UNL in the three ranges for certain situations/scenarios.

**Comment by Shadoga** [comment]
GND-UNL for RW Ranges 1 & 2, as well as the extension of RW Range 2 to north of  Rognan city, as well as any life fire northbound and above 3000 FT AMSL are regarded to be inacceptable options for 176th for RW Ranges inside the TMA.

The dreaded Basic CAS Range north of the field up to UNL already denies to ATCOs a usable radar pattern in the North.

Rotor-wing life fire combat activity shooting northbound towards the southern radar pattern and intermediate-to-final approach procedures RWY25, combined with red ground units shooting upwards above 3000 FT at other blue ENBO traffic overflying the RW Ranges cannot earnestly be a thing.

176th ATCOs will work (with slight modifications) with the lower limits of the real TMA airspace "D non-CTR" as minimum vectoring altitudes in LotATC. If you look at those altitudes in the chart below, you will see where RW Ranges inside the TMA's southern part can have top levels of max. 3000 FT, 4000 FT and 6000 FT AMSL (500 FT below TMA airspace "D non-CTR").

![ENBO TMA](https://github.com/132nd-vWing/TRMA/assets/168900540/93e46a57-59ec-44cf-bb29-96cc81e30007)


With the Basic A-G Range in the southwestern corner of the TMA up to UNL impeding the southern radar pattern and the intermediate-to-final approach procedures RWY07, I would like to request on behalf of 176th that the northern boundary of Basic A-G Range is adjusted to extend along the northern boundary of RW Range 1, not futher to the North.

The TMA boundaries in the .miz are quite off, here are the correct real world TMA coords as they will most likely shortly be requested by the ENBO TMA Project team:
665700N 0130000E
672500N 0130000E
674000N 0135500E
672728N 0160630E
follow along yellow DCS ME boundary line between Norway & Sweden
665451N 0160235E
665000N 0145000E

![ENBO TMA DCS ME](https://github.com/132nd-vWing/TRMA/assets/168900540/b5704a6e-1b5b-4003-8ce8-19f4ebbde36c)

**Comment by 132nd-Erro** [comment]
Keep in mind the there will not be live fires northbound every event, nor will the CAS range to the north be activated every event. Will there be radar vectoring and radar patterns every event? Whereas I am not a 176th controller, I would imagine different situations and scenarios of range activations and block assignments creates a challenging variation in the control, rather than a blocker for performing control?

I am open to suggestions of other suitable areas for helicopters to operate, but they must be within the specified range and offer the terrain features (coastline, fjords, valleys) for us to practice our low level tactics. Up until yesterday, I forgot to consider an urban area inside the ranges, and Neck's proposal to incorporate the eastern part included Rognan. Given the EA state of Kola, we don't know where the next (somewhat dense) urban area will be, but from the current map layout it seems to be Mo i Rana roughly 58 nm to the south, inside the A-G range. To access this area, we would require a FARP somewhere and would operate isolated from the rest of the wing, much like we do today.

**Comment by Shadoga** [comment]
Some intermediate ENBO TMA Project Results as of 2024-06-04:

1. Recommendation for using real CTR airspace lateral dimensions.
Corner Points:
672530N 0141800E
671925N 0144800E
671300N 0145000E
670900N 0140500E
671600N 0135500E

2. Recommendation to use ENBO CTR vertical dimensions raised unrealistically to GND-2500 FT MSL.

3. Recommendation to use the realworld lateral dimensions of ENBO TMA:
Corner Points:
665700N 0130000E
672500N 0130000E
674000N 0135500E
672728N 0160630E then
follow along yellow DCS ME boundary line between Norway & Sweden until
665451N 0160235E
665000N 0145000E and close with first coordinate

4. Recommendation to use the realworld vertical dimensions of ENBO TMA:
GND-FL195

5. Any AAR anchors above the TMA are welcome, even if they use some of the upper TMA levels.
Recommendation for the lowest Tanker Base Level to be FL180 or above, if possible.
Tankers should be individually de-spawnable and re-spawnable by F10 menu script.

**Comment by 132ndNeck** [comment]
Version 2.3 available:
- Updated TMA and CTR per request to have real world coordinates
- Updated RW ranges
- Extended SEAD/DEAD range EAST
- Added red aggressor jets at Bodø

Link: https://cloud.132virtualwing.org/s/dwnf5mrYFDR2B6x

**Comment by 132nd-Erro** [comment]
@132ndNeck that looks great, thank you very much. As a final and minor request to this version, if possible, is to extend R2 slightly to the east so that the southern valley from the lake is included (marked with yellow highlighter to the S-E of the first image), which can serve as an attack route for different scenarios. 

Suggestion is to either extend the northern border point to the east. 
![image](https://github.com/132nd-vWing/TRMA/assets/101829260/e995af4a-5cdd-4c4e-873f-ddbc3ab2dce0)

Or move both north and south border points to the east, but not as far east. 
![image](https://github.com/132nd-vWing/TRMA/assets/101829260/694e38a0-2277-46ee-93e1-d8962b18260a)

**Comment by 132nd-Erro** [comment]
Did a brief test flight while checking some SOP stuff, ranges are good but the location of HAMMERHEAD was too tight to serve as a spawn location. It does not facilitate ground or hover taxi per SOP when multiple flights are starting up, but instead it forces air taxi (which is not always possible depending on loadout). The location of GUNSLINGER allows for all modes of taxi and flights do not interfere with each other when taxiing.

**Comment by Shadoga** [comment]
Sad to see that none of the request I made with 108th training requirements in mind made it into the latest version of the TRMA.miz v2.3
We can just hope that the input is still under discussion in the Ranges Project group.
If there are any so far projected other ranges envisioned for the requested usages, please share the vision in order to be tested on Sunday.
Otherwise I'd like to stress the following requests:
- Request AAR anchors in the Southwest of ENBO for the blue water ranges.
- Request Feet Wet and Feet Dry BFM Ranges close to the TMA, e.g. east of the southern A-G Range, south of RW Range 2.
- Request the CVOA to be moved slightly north-northeast to be located west of Lofoten Islands group, in order to make room for AAR anchors and a smaller BFM Range over water, and to shorten transit to the bases and ranges way up to the northeast.
- Most important of all for 108th:
Request at least one A-A Drone Range over land in order to practice against radar ground clutter.
This is a biggie, because right now in ATRM MOA-1, we have the option depending on approach angle to work our radars against the A.I. MG29 Drones either against the Gulf's water surface or against the ground clutter from the coast of Qatar.
Having all A-A Drone Ranges exclusively over blue water will deny 108th RIOs practicing one of their most important skills: look-down radar work and target acquisition against ground clutter. 

![TRMA MIZv23 Feedback](https://github.com/132nd-vWing/TRMA/assets/168900540/60d4b1ae-c044-4770-80ca-5cda24c1ac44)

**Comment by 132ndEvo** [comment]
I don't see a need to create a specific BFM zone and move the carrier. The ranges at sea should be able to create drones and sea units or be empty, depending on the training requirements.

Also, I know that step 2 is deciding the tanker track locations. I think we should try to plan their locations a little bit. I did some testing:

- Tanker speed 300 knots ( We have had a request that the tankers should cruise faster since that helps with controllability and is 
what they do in real life.)
- Because the bank angle is limited while tanking, the width of the tanker is quite large. 
- With a 30nm straight run (ordinary in big wing tankers), the area recommended for a tanker track is as follows.

![image](https://github.com/132nd-vWing/TRMA/assets/65813756/fcad1bb5-95dc-4e35-a730-ddf1ce490de4)

The size is large, but it will ensure no tanker enters a range (especially the A2A ranges)
We can mitigate the size of the tanker tracks by keeping the carrier where it is and putting the tankers above the carrier (the carrier's range size is perfect). If we limit the ceiling of the carrier's airspace to FL200 (carrier ops do not require higher) Then the tankers overhead will support the carrier, all oceanic ranges, and the ranges just north and south of Bodo. We can even move the track to the east to be closer to Bodo and it will not disrupt any training since only Bodo and the carrier's airspace will be infringed on.  

In summation:

- Do not limit ranges to a specific purpose. (unless the adversary requires a lot of room, i.e., SA-10 or Type 52) 
- Keep the carrier where it is. 
- Place tanker tracks above the carrier to support carrier ops, oceanic ranges, and the ranges near Bodo.

**Comment by 132nd-Thatch** [comment]
Looking at where the carrier is, the Carrier Air Wing will be 122NM from Bodo (at the furthest) and the Western side of most ranges. It will take longer to get elsewhere. At the moment that would take Hornets 15 minutes cruising at M0.70 not taking into account the climb or CASE III departure. So probably 30-40 minutes just to get to the ranges and back. In the Gulf map, the carrier was approx 100-120 miles from the SEAD ranges and we were always minimum time over there or going back for gas then again to the range. Longest sorties were to there.  Looking at the Kola map that would be standard.

We need a few Hornets at Bodo parked together, spoke to Evo and think 2 x Hornets at G01,G02, 2 x Hornets G06,G07 and a four-ship at G10,G12,G13,G14 would cover our options for getting to the ranges to have more time there.

Regarding the RedAir at Bodo. Are they used that often to take all that space? Can they go on the North side until new bases open up?

Agree with Shadoga about the SW AAR area, also need one to east of the SEAD/DEAD range but outside the Drone range Shadoga suggested.

**Comment by 000Ready000** [comment]
Would it be possible to carve out an extra area for A/G the east of Bodo? 
Would love to practice low level in between snowy mountains, close to that high peak.
This area seems unoccupied at the moment.

![image](https://github.com/132nd-vWing/TRMA/assets/98852210/8fa9eef9-05ed-4f67-9b1f-328b279b8793)

**Comment by Gabykhan** [comment]
> Sorry if I am late in the process, but would it be possible to carve out an extra area for A/G the east of Bodo? Would love to practice low level in between snowy mountains, close to that high peak. This area seems unoccupied at the moment.
> 
> ![image](https://private-user-images.githubusercontent.com/98852210/340226935-8fa9eef9-05ed-4f67-9b1f-328b279b8793.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg2MzgwNTIsIm5iZiI6MTcxODYzNzc1MiwicGF0aCI6Ii85ODg1MjIxMC8zNDAyMjY5MzUtOGZhOWVlZjktMDVlZC00ZjY3LTliMWYtMzI4YjI3OWI4NzkzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjE3VDE1MjIzMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTFmNTBiZWExMjlmMWQ5NTU2OWU3YjIwOGFkZTc0MGM2OWM1ZmUyODU1OTYxM2E4MzVkYzM0OTUyYjgzZWM5MzgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.LnZ6CX21NXfzPTVGg2r9Gm6KVzLF3ljQCqo6u4kDd_A)

After testing the "Basic A/G Range" on June 16th event, I concur with this proposal in principle. The current basic A/G range is nice (cf. dedicated thread later) but is mostly over the water. This kind of landscape is not representative of the general situation, and having a basic range over land would be a nice addition.
However, I don't agree with the proposed location. We should not add up ranges averywhere around Bodo, otherwise when all ranges are active we are more of less stuck in the middle and it will become painful to depart from / return to the airfield.
We could add "practise targets" in the southern A/G range, for example on the eastern part of it.

Now regarding current "Basic A/G Range", a closer study shows that it is too small, especially because all the targets are located on the SE part of the range.
If we take a 1 minute / 9 Nm run from IP to target, which is already somewhat short, then we can only use N and W of the range.

From this simple remark, I suggest slightly increasing the range to the S and the E in order to have enough space to create patterns from any direction.

See attached picture and [.cf file](https://disk.yandex.com/d/1IfuMjcy44UeGw) for a proposal.

![Proposed Basic AG Range](https://github.com/132nd-vWing/TRMA/assets/141456266/9ece2304-ecf9-4880-90e1-b87ce390c828)

**Comment by 000Ready000** [comment]
> After testing the "Basic A/G Range" on June 16th event, I concur with this proposal in principle. The current basic A/G range is nice (cf. dedicated thread later) but is mostly over the water. This kind of landscape is not representative of the general situation, and having a basic range over land would be a nice addition. However, I don't agree with the proposed location. We should not add up ranges everywhere around Bodo, otherwise when all ranges are active we are more of less stuck in the middle and it will become painful to depart from / return to the airfield. We could add "practise targets" in the southern A/G range, for example on the eastern part of it.

Ahh I see the proposed corridor from Bodo to and from the east in the earlier drawing. Makes sense to keep this clear. My main point is to have an over-land range with hopefully some white peak mountains with some valleys very close by and to be able to fly through, use for masking, tight low level maneuvering and enjoy (also for the added visuals during sorties). Will check the east of the southern A/G range.

**Comment by 132nd-Thatch** [comment]
Jokkmokki airbase is much more developed than other road bases with runway and taxiway infrastructures, it has buildings as well.
* We do need ATIS or take Bodo because cross wind landings are tough on landing to keep  on the road.
* TACAN worked well
* We need an airfield/roadway map as there are a few tracks going off the taxiways through the trees that end up nowhere, I had to use F2 to taxy between each.  No boulders or grass on the runways or taxiways.
* It has a large landing area and three small road runways, no problem operating from them but in bad weather and strong winds the small ones might be a challenge.
* The lights along the side of the runways were working and on.
* Need to move the Jokkmokki FOB label to the actual place not West of it.
* It took 16 minutes to Jokkmokki from Bodo.

Range 23, all SAM sites worked. The SA-6 sites didn't lock me up until I was within 5NM of them. SA-15 I didn't get close too before being locked up.

FLOT is in a good position and where I descended though cloud it was where it met a river and high ground on the northern banks, could use the NW-SE river as a reference for the FLOT line as it is very easy to see.

N65 34 67 E020 05 24 could be a good area for searching for vehicles on dirt track roads.
N65 45 69 E020 17 55 could be a good area for a RED training camp complex.
N65 54 46 E020 19 16 could be a good reference point for CAS training for off road vehicles etc as it is a large circle with defined structures inside it.

**Comment by 132nd-Erro** [comment]
Bumping this https://github.com/132nd-vWing/TRMA/issues/15#issuecomment-2155302812

**Comment by 132ndNeck** [comment]
> Bumping this [#15 (comment)](https://github.com/132nd-vWing/TRMA/issues/15#issuecomment-2155302812)

Like this?
![bilde](https://github.com/132nd-vWing/TRMA/assets/26002570/0615e7f4-36f5-44be-83b9-d49502bc29c8)

**Comment by 132nd-Erro** [comment]
Looks great, thank you

**Comment by Gabykhan** [comment]
Small remark regarding current layout: closest alternate airfield from ENBO is inside a training range (Scenario range C).
Most likely we'll have additional airfields in the future, but we should make sure that one of them close to ENBO is not inside any range.

**Comment by 132ndNeck** [comment]
> Small remark regarding current layout: closest alternate airfield from ENBO is inside a training range (Scenario range C). Most likely we'll have additional airfields in the future, but we should make sure that one of them close to ENBO is not inside any range, so that it can become an alternate airfield (knowing that IRL 2 alternates are often required both for take-off and landing).

Yes, both Evenes and Andøya should be outside any active ranges.

**Comment by Shadoga** [comment]
Andoya is just inside the current Range 21 boundary, but that should not constitute any problem.
I suggest to already install the real world CTR lateral boundaries for Evenes and Andøya into the .miz.
In order to show where they will be located and to prevent placement of any other training installations or vehicles there.
Can provide coordinates if desired.

**Comment by 132ndEvo** [comment]
Can we add a large oceanic range to the north of the A/A drone range? The type 52C shot down aircraft at 70nm. It would be nice to have a large range (100 x 100 nm) to the north to practice against these difficult units. Thanks

**Comment by 132ndNeck** [comment]
> Can we add a large oceanic range to the north of the A/A drone range? The type 52C shot down aircraft at 70nm. It would be nice to have a large range (100 x 100 nm) to the north to practice against these difficult units. Thanks

Yes, I planned for this, but did not get the time before I left for vacation. But absolutelly. A "blue ocean" scenario range is intended north of the already established ranges. Can be combined with a red CVN as required. 
But will have a more detailed look when I get back from vacation

**Comment by 132ndNeck** [comment]
> Andoya is just inside the current Range 21 boundary, but that should not constitute any problem. I suggest to already install the real world CTR lateral boundaries for Evenes and Andøya into the .miz. In order to show where they will be located and to prevent placement of any other training installations or vehicles there. Can provide coordinates if desired.

Yes, good point. And happy to add the CTR boundaries if you have the coordinates at hand. Will implement when I am back from vacation.

**Comment by Shadoga** [comment]
ENAN Andoya CTR GND-A2500 FT AMSL
Corner Points:
693500N 0160000E
692600N 0164000E
690700N 0164000E
690200N 0160500E
692900N 0153100E

ENEV Evenes CTR GND-A2500 FT AMSL
Corner Points:
684400N 0163000E
684400N 0165000E
683341N 0165000E
682852N 0165611E
682624N 0165611E
682200N 0165000E
682058N 0163656E
682200N 0161500E

ENDU Bardufoss CTR GND-A4500 FT AMSL
Corner Points:
691500N 0175300E
690500N 0191500E
685500N 0190500E
690500N 0174000E

ENTC Tromso CTR GND-A4500 FT AMSL
Corner Points:
695800N 0184400E
695400N 0193000E
692615N 0190300E
692800N 0182600E

ENNA Banak CTR GND-A3500 FT AMSL
Corner Points:
702000N 0244000E
702000N 0252000E
694500N 0252000E
694500N 0244000E
695236N 0244000E
700600N 0242646E
701000N 0242646E
701430N 0244000E

ENKR Kirkenes CTR GND-A2500 FT AMSL
Corner Points:
694149N 0290806E
695000N 0294000E
695200N 0302500E
694000N 0304000E
694010N 0300906E
southwards along the border between Norway and Russia to 693930N 0300509E
692904N 0292010E
westwards along the border between Norway and Finland to first coordinate (694149N 0290806E)

ENAT Alta CTR GND-A3500 FT AMSL
Corner Points:
700800N 0230000E
700800N 0233600E
700200N 0235100E
695400N 0235100E
694546N 0233702E
694520N 0233124E
694800N 0231100E
695100N 0225800E
700200N 0225000E

So far for the Norwegian TWR CTRs that might appear sooner or later or never in the Kola Map.
ENNA is already there, ENAN and ENEV are announced, as far as I know. Not sure about any others.
Smaller aerodromes like e.g. ENLK Leknes don't feature TWR CTRs (airpace class "D CTR") but are uncontrolled and have Traffic Information Zones (TIZ).

Going to start searching next for the CTR coordinates of the Swedish and Finnish bases already implemented or announced.

**Comment by Shadoga** [comment]
Found something:

ESNQ Kiruna CTR GND-A3100 FT AMSL
Corner Points:
680054N 0202744E
675754N 0204244E
674625N 0203344E
673725N 0201444E
674025N 0195844E
675154N 0200704E

EFRO Rovaniemi CTR GND-A2000 FT AMSL
(we should increase to minimum A2500 FT AMSL due to HPMA pattern altitude of 2100 FT AMSL)
Corner Points:
664733N 0255218E
664140N 0261148E
663520N 0261613E
661950N 0254634E
662921N 0251644E

So, for the time, I recommend we implement at least ENNA, ESNQ & EFRO CTRs in the .miz because these aerodromes already exist in the Kola Map, plus add ENAN and ENEV CTRs for future reference.
Fingers crossed we can get more airbases released by ORBX soon...

**Comment by Shadoga** [comment]
Found this regarding planned airbases for Kola:
https://www.google.com/maps/d/viewer?mid=1k-Km6-IscXX3Uh8TvugXHwZrRQGFqr8&ll=67.93113052409831%2C24.24098406338407&z=6

So it should not hurt to implement ENDU Bardufoss, ENTC Tromso and ENKR Kirkenes CTRs as well, in order to show where they will be located and to prevent placement of any other training installations or vehicles there.

**Comment by 000Ready000** [comment]
Did we already put A-10's on Kiruna and Jokkmokk? If not, could we request 3 hogs on both locations? Would love to try R22, but it is a long ride from Bodo.

**Comment by Shadoga** [comment]
EFKT Kittilae CTR GND-A3300 FT AMSL
Corner Points:
675626N 0250229E
673134N 0251740E
672809N 0243941E
675258N 0242352E
